src/mudblazor/MudBlazor.Template/Components/Pages/Home.razor

```diff
@@ -2,7 +2,6 @@
 
 <PageTitle>Home</PageTitle>
```

Comment:
I think we could do a better job on the Home page by customizing the text based on the project type.